tests/stress/main_test.go

```diff
@@ -27,27 +26,19 @@ func TestMain(m *testing.M) {
 
 func setup() {
 	flag.Parse()
-
-	log.Level = logrus.ErrorLevel
-	if *debug {
-		log.Level = logrus.DebugLevel
-	}
-
-	logutils.SetGlobalLogger(log)
-
+	logger := logging.NewLogWriter(*debug)
+	log.SetOutput(logger)
 	if !*dry {
 		if err := filepath.Walk("cases", deleteStateFiles); err != nil {
-			fmt.Println("Error occurred during setup: ", err)
-			os.Exit(1)
+			log.Fatalf("[ERROR] Error occurred during setup: ", err)
```

Comment:
Continuing on from the Dave Cheney mindset, why have this go to logs?

Reply:
I could instead do something like `log.Printf(err.Error())`?

Reply:
Or just do what the code did previously

Reply:
:white_check_mark: